---
title: "LtrDetector: A tool-suite for detecting long terminal repeat retrotransposons de-novo"
collection: publications
permalink: /publication/2019-06-03-paper-LtrDetector
excerpt: 'LtrDetector uses techniques inspired by signal-processing for de-novo detection of long terminal repeat retrotransposons.'
date: 2019-06-03
venue: 'BMC Genomics'
paperurl: 'http://academicpages.github.io/files/LtrDetector.pdf'
citation: 'Valencia, J.D., Girgis, H.Z. LtrDetector: A tool-suite for detecting long terminal repeat retrotransposons de-novo. *BMC Genomics* **20**, 450 (2019). [](h    ttps://doi.org/10.1186/s12864-019-5796-9)'
---
Long terminal repeat retrotransposons are the most abundant transposons in plants. They play
important roles in alternative splicing, recombination, gene regulation, and defense mechanisms. Large-scale
sequencing projects for plant genomes are currently underway. Software tools are important for annotating long
terminal repeat retrotransposons in these newly available genomes. However, the available tools are not very sensitive
to known elements and perform inconsistently on different genomes. Some are hard to install or obsolete. They may
struggle to process large plant genomes. None can be executed in parallel out of the box and very few have features
to support visual review of new elements. To overcome these limitations, we developed LtrDetector, which uses
techniques inspired by signal-processing.

Recommended citation: Valencia, J.D., Girgis, H.Z. LtrDetector: A tool-suite for detecting long terminal repeat retrotransposons de-novo. *BMC Genomics* **20**, 450 (2019). [https://doi.org/10.1186/s12864-019-5796-9](https://doi.org/10.1186/s12864-019-5796-9)
